src/routes/index.jsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
+export default function Index() {
+	return (
+		<p id="zero-state">
+		This is a demo for React Router.
+		<br />
+		Check out{" "}
+		<a href="https://reactrouter.com">
+				the docs at reactrouter.com
+		</a>
+		.
+		</p>
+	);
+}
```

Comment:
- [ ] I do not think this file is related to this particular project, kindly remove it to make your project more professional.